apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx

```diff
@@ -30,14 +31,17 @@ const activityStatusLabels: Partial<Record<ActivityStatus, string>> = {
   [ActivityStatus.CANCELLING]: 'Cancelling',
   [ActivityStatus.FAILED]: 'Failed',
 }
-function getTitleStatus(activityDerivedState: ActivityDerivedState | null): string {
+function getTitleStatus(activityDerivedState: ActivityDerivedState | null, safeWallet: GnosisSafeInfo | undefined): string {
+
   if (!activityDerivedState) {
     return ''
   }
 
   const prefix = activityDerivedState.isOrder ? 'Order' : 'Transaction'
-  const postfix = activityStatusLabels[activityDerivedState.status] || 'Submitted'
-
+  const postfix =
+    activityStatusLabels[activityDerivedState.status] || (safeWallet && safeWallet.threshold > 1)
+      ? "Order Submitted, but won't be processed until the transaction is signed by all your Safe{Wallet} signers."
+      : 'Submitted'
```

Comment:
Awesome. I think we might want to refine these texts. @MindyCoW basically 2 cases:
- **Normal case**: we just say `Submitted`
- **Orders for safes with threshold greater than 1:** `Order Submitted, but won't be processed until the transaction is signed by all your Safe{Wallet} signers`


I would suggest to iterate on top of this PR and just present it nicer. 
One suggestion is to show also `Submitted` but then show below an icon showing some key+people in it with a test. "The order has been placed, but it won't be [OPEN] until all signers of the multisig wallet signs and execute the transaction."

Reply:
Safe calls signatures "confirmations":

<img width="373" alt="image" src="https://github.com/cowprotocol/cowswap/assets/104489816/8cce1112-97cb-46a5-a38a-fe2896f7ee8f">

So perhaps we should show: 

- **1:1 Safe:** `Submitted`
- **Multisig Safe:** `Order submitted. Awaiting confirmation from all signers.`